integration/test-integration/src/test/resources/testng-cc-with-apim.xml

```diff
@@ -63,6 +63,12 @@
         </classes>
     </test>
 
+    <test name="Test SOAP API invocation" parallel="false">
+        <classes>
+            <class name="org.wso2.choreo.connect.tests.testcases.withapim.soap.SOAPApisTestCase"/>
+        </classes>
```

Comment:
Shall we move this into the group that starts in line 78? we can add this right before the websocket tests

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/17a62e5eac8c71ad987bbbae2a42481ac4979fa6